landingzones/tests/test_views_api.py

```diff
@@ -221,3 +221,37 @@ def test_get_locked(self):
         response = self.request_knox(url)
         self.assertEqual(response.status_code, 200)
         self.assertEqual(json.loads(response.content)['status_locked'], True)
+
+
+class TestLandingZoneUpdateAPIView(TestLandingZoneAPIViewsBase):
+    """Tests for LandingZoneUpdateAPIView"""
+
+    def test_patch(self):
+        """Test LandingZoneUpdateAPIView patch() as zone owner"""
+        url = reverse(
+            'landingzones:api_update',
+            kwargs={'landingzone': self.landing_zone.sodar_uuid},
+        )
+        data = {
+            'assay': str(self.assay.sodar_uuid),
+            'description': 'New description',
+            'user_message': 'New user message',
+        }
+        response = self.request_knox(url, method='PATCH', data=data)
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(
+            json.loads(response.content)['description'], 'New description'
+        )
+        self.assertEqual(
+            json.loads(response.content)['user_message'], 'New user message'
+        )
+
+    def test_patch_title(self):
+        """Test updating title with patch() (should fail)"""
+        url = reverse(
+            'landingzones:api_update',
+            kwargs={'landingzone': self.landing_zone.sodar_uuid},
+        )
+        data = {'assay': str(self.assay.sodar_uuid), 'title': 'New title'}
+        response = self.request_knox(url, method='PATCH', data=data)
+        self.assertEqual(response.status_code, 500)
```

Comment:
This should return 400 (bad request), not 500 (internal server error). Same with other fields which are not allowed to be edited.